paying_for_college/models.py

```diff
@@ -243,6 +266,41 @@ def primary_alias(self):
             return 'Not Available'
 
 
+class Notification(models.Model):
+    """record of a disclosure verification"""
+    institution = models.ForeignKey(School)
+    oid = models.CharField(max_length=40)
+    timestamp = models.DateTimeField()
+    errors = models.CharField(max_length=255)
+    email = models.CharField(max_length=255, blank=True)
+
+    def __unicode__(self):
+        return "{0} {1}".format(self.oid, self.institution.primary_alias)
+
+    def notify_school(self):
+        payload = {
+            'oid': self.oid,
+            'time': self.timestamp.isoformat(),
+            'errors': self.errors
+        }
+        school = self.institution
+        # we prefer to use endpount notification, so use it first if existing
+        if school.contact:
+            if school.contact.endpoint:
+                requests.post(school.contact.endpoint, data=payload)
+                return "school notified via endpoint"
+            elif school.contact.contact:
+                send_mail("CFPB disclosure notification",
```

Comment:
excellent idea; grazie. 
the current PFC tool is using send_mail; i'll check settings.